### DISCUSSION TITLE
Add FAQ section to the webpage

### DISCUSSION THREAD

**Discussion by ibrahimraimi** [chryz-hub/chryz-hub.github.io#262] [Ideas]
**Description**
  Add frequently asked questions to the webpage.

A brief description of the ideas:

   A list of questions and answers.
Example:
```
Question: How do I join chryz-hub?
Answer: It is best to  join Discord to start collaborating with the inclusive chryz-hub community.
```

```
Question: Who is chryz-hub for?
Answer: Whether you are just starting your journey into tech or you are a  professional, you will benefit from joining chryz-hub. 
Open Source is about collaboration.  Members include developers, testers, DevOps,  technical writers, UI and UX designers.
```

**Comment by namangupta1399** [comment]
Yeah. Sounds good to me.
@chryzcodez , what do you think?

**Comment by chryzcode** [comment]
Okay...wow, thanks for the idea..

We can do that. 👍

**Comment by Evavic44** [comment]
Great Idea @IbrahimOlayinka I was looking online, saw some designs for the FAQ page. Will share on discord.

**Reply by chryzcode** [reply]
Please share some😉

**Reply by ibrahimraimi** [reply]
Common questions are

1. What is open Source
2. What is Chryz-Hub?
3. Who is Chryz-Hub for?
4. How do I join Chryz-Hub?
5. Why join Chryz-Hub
6. I am new to coding and / or Open Source, can I still join?
